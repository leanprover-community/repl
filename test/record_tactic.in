{"cmd": "example : 1=1 := by sorry", "record": false}

{"cmd": "example : 1=1 := by sorry", "record": true}

{"cmd": "example : 1=1 := by sorry"}

{"tactic": "rfl", "proofState": 0}

{"tactic": "rfl", "proofState": 1}

{"tactic": "rfl", "proofState": 2}
