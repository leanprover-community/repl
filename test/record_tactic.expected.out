{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 20},
   "goal": "⊢ 1 = 1",
   "endPos": {"line": 1, "column": 25}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}]}

{"sorries":
 [{"proofState": 1,
   "pos": {"line": 1, "column": 20},
   "goal": "⊢ 1 = 1",
   "endPos": {"line": 1, "column": 25}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"sorries":
 [{"proofState": 2,
   "pos": {"line": 1, "column": 20},
   "goal": "⊢ 1 = 1",
   "endPos": {"line": 1, "column": 25}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

{"proofStatus": "Completed", "proofState": 3, "goals": []}

{"proofStatus": "Completed", "proofState": 4, "goals": []}

{"proofStatus": "Completed", "proofState": 5, "goals": []}

